bootstrap/terraform/main.tf

```diff
@@ -209,3 +209,22 @@ resource "kubectl_manifest" "kubernetes_provider" {
 
   depends_on = [module.kubernetes-addons]
 }
+
+#---------------------------------------------------------
+# Crossplane Helm Provider deployment
+# Creates ProviderConfig name as "helm-provider-config"
+#---------------------------------------------------------
+data "kubectl_path_documents" "helm_provider_manifests" {
+  pattern = "${path.module}/crossplane-providers/helm-provider.yaml"
+  vars = {
+    package-version = "crossplanecontrib/provider-helm:v0.12.0"
+    service-account = "crossplane-provider-helm"
+  }
+}
```

Comment:
Helm provider is already being installed in this main.tf here: https://github.com/awslabs/crossplane-on-eks/blob/e3893764d305aaa583e7618282668239d039afa6/bootstrap/terraform/main.tf#L226-L232.